docker-compose.yaml

```diff
@@ -18,7 +18,7 @@ services:
     image: dpage/pgadmin4:snapshot
     container_name: postgres-admin
     ports:
-      - "5050:80"
+      - "80:80"
```

Comment:
Good to know. I'll rollback to the `5050` post since the Dockerfile uses the `8080` , just  to prevent conflicts if anyone wants to use the Dockerfile in local.